.env

```diff
@@ -5,5 +5,6 @@ REACT_APP_CHAIN_2_CUSTOM_TYPES_URL=https://raw.githubusercontent.com/paritytech/
 REACT_APP_CHAIN_2_SUBSTRATE_PROVIDER=wss://wss.millau.brucke.link
 
 REACT_APP_LANE_ID=0x00000000
+REACT_APP_ACCOUNTS_INFORMATION_UPDATE_FREQUENCY=30000
```

Comment:
What's the point of making this configurable? This is just noise/boileplate imho. The UI should just work correctly without any need of fine-tuning.

Reply:
i think it is a nice feature to leave this configurable because hypothetically a certain team wants to implement their own pair UI, they will have the chance to customize this value as most of the other values we leave to configure from the .env.

Reply:
Honestly, I can't imagine any use case in which this would be useful to make this particular parameter configurable. It's basically like letting the "Submit" button to have customizable label via `.env` variables "because hypothetically...".
The choice to make this parameter customizable (and not others) is completely arbitrary and not based on any use cases we have considered. Either let's hardcode that, or if your intention is to go extra mile, let's make it based on the block time (i.e. refresh every four blocks or 30 seconds, whatever comes first).

Reply:
OK. We can definitely go for the four blocks approach if you like. But if we do this, my recommendation would be to remove the `setInterval` mechanism and just leave the blocks number count since i don't see much sense on keeping both. What do you think?

Reply:
>  (i.e. refresh every four blocks or 30 seconds, whatever comes first).

OK. Working on this.